packages/fxa-settings/src/pages/Signin/CompleteSignin/index.tsx

```diff
@@ -19,20 +20,20 @@ export type CompleteSigninProps = {
   linkStatus: LinkStatus;
 };
 
+export const viewName = 'complete-signin';
+
 const CompleteSignin = ({
   brokerNextAction,
   isForPrimaryEmail,
   linkStatus,
 }: CompleteSigninProps & RouteComponentProps) => {
-  usePageViewEvent('complete-signin', {
-    entrypoint_variation: 'react',
-  });
+  usePageViewEvent(viewName, REACT_ENTRYPOINT);
   const navigate = useNavigate();
   const linkType = 'signin';
   const [error, setError] = useState<string>();
   // there is no valid view in the `complete_signin` mustache. It completes your signin,
   // and then redirects you according to the broker method. The default is to `signin_verified`.
-  // I think it makes sense to keep this logic in this page (as opposed to housing it a level up,
+  // I think it makes sense to keep this logic in this cd age (as opposed to housing it a level up,
```

Comment:
👀  typo?

Reply:
The heck, I literally have no idea how this happened, thanks. 😆

Reply:
@pdehaan, finding the bugs that matter(tm).